scripts/whitelist_utils.js

```diff
@@ -0,0 +1,143 @@
+const Web3Utils = require("web3-utils");
+
+/* Optional initial Whitelisted Nodes */
+let envInitialWhitelistedNodes = process.env.INITIAL_WHITELISTED_NODES;
```

Comment:
for consistency should all 3 be initialised here?

Reply:
@macfarla thx for picking it, I was meant to move them ... they are now moved inside respective methods.